src/journal/journalctl.c

```diff
@@ -2408,6 +2434,11 @@ static int run(int argc, char *argv[]) {
         if (r < 0)
                 return log_error_errno(r, "Failed to add filter for syslog identifiers: %m");
 
+        r = add_exclude_identifier(j);
+        if (r < 0) {
```

Comment:
no brackets around single line if blocks

Reply:
Thank you! Done